host-controller/src/main/java/org/jboss/as/domain/controller/resources/DomainDeploymentResourceDefinition.java

```diff
@@ -125,4 +131,26 @@ protected void registerAddOperation(ManagementResourceRegistration registration,
         registration.registerOperationHandler(addDefinition, handler);
     }
 
+    static class ServerGroupDomainDeploymentResourceDefinition extends DomainDeploymentResourceDefinition {
```

Comment:
Should be private

Reply:
I don't think being private here should make much difference, we had this discussion with David a number of years back on Elytron.  I believe even if you set this to private the resulting class still needs to be given package level visibility so it can be accessed by the class which contains it.

Most importantly nothing here is public so nothing will leak as API used by other packages.

Reply:
changed to private